internal/post/message.go

```diff
@@ -70,14 +70,24 @@ func (msg *Message) Bytes() []byte {
 		var sub io.Writer
 
 		if len(msg.PlainText) > 0 {
-			sub, _ = wr.CreatePart(textproto.MIMEHeader{
+			sub, err := wr.CreatePart(textproto.MIMEHeader{
 				"Content-Type":              []string{"text/plain; charset=UTF-8; format=flowed"},
 				"Content-Transfer-Encoding": []string{"quoted-printable"},
 			})
+			if err != nil {
+				return nil, err
```

Comment:
Let's also use error wrapping to be able to trace back to the location where the error happened.